homeassistant/components/fitbit/application_credentials.py

```diff
@@ -60,7 +60,10 @@ async def _post(self, data: dict[str, Any]) -> dict[str, Any]:
             resp.raise_for_status()
         except aiohttp.ClientResponseError as err:
             if _LOGGER.isEnabledFor(logging.DEBUG):
-                error_body = await resp.text() if not session.closed else ""
+                try:
+                    error_body = await resp.text()
+                except aiohttp.ClientError:
+                    error_body = ""
```

Comment:
Regarding the coverage warning: I verified coverage for all of this running tests with debug mode enabled (reproduces the issue and the fix fixes the issue)